web/src/message_list_view.ts

```diff
@@ -603,25 +542,13 @@ export class MessageListView {
         } else {
             last_edit_timestamp = message.last_edit_timestamp;
         }
```

Comment:
The if/else block above is where we handle the `local_edit_timestamp` for the `message_container.last_edit_timestamp` field.